internal/loadbalancebenchmark/config.go

```diff
@@ -0,0 +1,120 @@
+package loadbalancingbenchmark
+
+import (
+	"fmt"
+	"time"
+)
+
+type ClientGroup struct {
+	Cnt    int
+	Rps    int
+	LType  ListType
+	LUType ListUpdaterType
+}
+
+type LatencyConfig struct {
+	P50  time.Duration
+	P90  time.Duration
+	P99  time.Duration
+	P100 time.Duration
+}
+
+type ServerGroup struct {
+	Cnt           int
+	Type          MachineType
+	LatencyConfig *LatencyConfig
+}
+
+type TestConfig struct {
+	ClientGroup []ClientGroup
```

Comment:
Since this is the entry point to the benchmark, let's also add comments to these fields